src/systemtask/SystemTask.cpp

```diff
@@ -273,6 +277,14 @@ void SystemTask::Work() {
           // Remember we'll have to reset the counter next time we're awake
           stepCounterMustBeReset = true;
           break;
+        case Messages::OnChargingEvent:
+          // call the battery controller or use the MSG in DisplayApp to get the battery status ???
```

Comment:
Calling `batteryController.Update()` here is not necessary as it was already called on line 170, just after the message was received. It does however shows why we implemented this event OnChargingEvent so we can either keep it or comment that this event is there only to unlock the system task and that the battery level has already been updated a few lines above :)

But I think we can safely remove the comments as we don't need them to understand the code.
But of course, if you don't want us to forget about possible optimizations, feel free to create issues about them (batter and motion).